### PR TITLE
Added file lock notes.

### LINKED ISSUE #763: Allow extended cloud check to block and scan for up to (seconds) = 0 ?
Hello @aczechowski, Jose from IU here.

Can I assume "Allow extended cloud check to block and scan for up to (seconds)" = 0 would mean the extended cloud check to block and scan could take an infinite amount of time? If so, is there a default time out?

Sincerely,
Jose

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5215a151-f390-9123-5384-516ec3ed3078
* Version Independent ID: 62124728-8029-4565-79a6-88f842ad1e73
* Content: [Endpoint Protection antimalware policies - Configuration Manager](https://docs.microsoft.com/en-us/sccm/protect/deploy-use/endpoint-antimalware-policies#cloud-protection-service)
* Content Source: [sccm/protect/deploy-use/endpoint-antimalware-policies.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/protect/deploy-use/endpoint-antimalware-policies.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by ChrisKibble**
Added notes that the setting entered is in additional to a pre-set 10 second lock.  Fixes #763.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@jborsecnik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/protect/deploy-use/endpoint-antimalware-policies.md [hunk 1]
```diff
@@ -204,6 +204,9 @@ Cloud Protection Service enables the collection of information about detected ma
 
 **Allow extended cloud check to block and scan for up to (seconds)** - Specifies the number of seconds Cloud Protection Service can block a file while the service checks that the file is not known to be malicious.
 
+> [!NOTE]  
+> The number of seconds selected for this setting will be in addition to a 10-second timeout built in to the product.  Example: if you enter 0 seconds, CSP will be able to lock the file for up to 10 seconds.
```
Comment (aczechowski) [inline sccm/protect/deploy-use/endpoint-antimalware-policies.md]: A rewording suggestion for style:

```suggestion
> The number of seconds that you select for this setting is in addition to a default 10-second timeout. For example, if you enter 0 seconds, the Cloud Protection Service blocks the file for 10 seconds.
```

**Comment by PRMerger17** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/protect/deploy-use/endpoint-antimalware-policies.md (hunk 1 above)
Comment (ChrisKibble) [reply sccm/protect/deploy-use/endpoint-antimalware-policies.md]: Thanks, yours definitely sounds much better.

**Comment by mestew** [comment]
#sign-off

**Pull request merged**